doc/_scripts/gen_devicetree_rest.py

```diff
@@ -226,9 +226,11 @@ def load_base_binding():
     # nodes from node-specific properties.
 
     base_yaml = ZEPHYR_BASE / 'dts' / 'bindings' / 'base' / 'base.yaml'
+    base_includes = {"pm.yaml": os.fspath(ZEPHYR_BASE / 'dts' / 'bindings' / 'base'/ 'pm.yaml')}
+
     if not base_yaml.is_file():
         sys.exit(f'Expected to find base.yaml at {base_yaml}')
-    return edtlib.Binding(os.fspath(base_yaml), {}, require_compatible=False,
+    return edtlib.Binding(os.fspath(base_yaml), base_includes, require_compatible=False,
```

Comment:
@mbolivar-nordic can you review these `gen_devicetree_rest.py` changes?

Reply:
that is required because the base.yaml is including the new yaml in that directory.

Reply:
I think we should just add them to base.yaml, then.

Reply:
Never mind, I see you had a long discussion about this already that I didn't read at first. Sure; that seems fine as-is.